azure.env

```diff
@@ -0,0 +1,7 @@
+# Azure Open AI

```

Comment:
Best is to add this `azure.env` to your gitignore now, so that new changes to this file won't be committed accidentally.

Reply:
Yes was not sure about this possibilité for a file
I tried env param on windows but didnt work